### PR TITLE
Do not display the small video for the main speaker

### PR DESCRIPTION
**Pull request by imobachgs**
When a user is the main speaker, her/his small video should be stopped (replaced by the user icon).

### CODE REVIEW

File: src/components/Participant/Participant.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ function Participant({
   const dispatch = useDispatch();
   const videoRef = React.createRef();
   const cssClassName = `Participant ${focus === 'user' ? 'focus' : ''}`;
-  const showVideo = video || isLocalScreen;
+  const showVideo = (video || isLocalScreen) && !focus;
```
Comment (mvidner) [inline src/components/Participant/Participant.js:42]: this is simple enough but I'd like to glue some JSDocs for the Participant props onto it :)

File: src/components/Participant/Participant.js [hunk 2]
```diff
@@ -39,7 +39,7 @@ function Participant({
   const dispatch = useDispatch();
   const videoRef = React.createRef();
   const cssClassName = `Participant ${focus === 'user' ? 'focus' : ''}`;
```
Comment (mvidner) [inline src/components/Participant/Participant.js:41]: ... because here it looks like `focus` is not the simple boolean the one-liner change would suggest

**Review by mvidner — CHANGES_REQUESTED** [review]
Please fill in at least the `focus` property. What does 'user' mean?

```js
/**
 * @param props {object}
 * @param props.id {object} not the superego
 * @param props.username {}
 * @param props.isPublisher {}
 * @param props.isLocalScreen {}
 * @param props.streamReady {}
 * @param props.focus {}
 * @param props.speaking {}
 * @param props.video {}
 */
function Participant({

```

File: src/components/Participant/Participant.js (hunk 2 above)
Comment (imobachgs) [reply src/components/Participant/Participant.js:41]: Yes, you are right, it is wrong. Let me fix it.

**Review by mvidner — APPROVED** [review]

**Pull request merged**